src/components/ReviewCard.jsx

```diff
@@ -1,19 +1,53 @@
+import React, { useState } from "react";
+import API from "../axios/API";
+import IndvReview from "./IndvReview";
+
 const ReviewCard = ({ data }) => {
```

Comment:
I like the logic but I would recommend renaming this component as it seems to be for a multiple reviews rather than just 1 card? Also could links along with routes be used to make your url more reflective of what is being shown?

May also be beneficial to have the individual review card extracted out into another component for re-use